src/tools/vmap4_extractor/vmapexport.cpp

```diff
@@ -488,8 +488,8 @@ int main(int argc, char ** argv)
 #endif
                     ))
             success = (errno == EEXIST);
-
-    int FirstLocale = -1;
+    int defaultFirstLocale = -1;
```

Comment:
please declare this as const so it doesn't get modified by accident in the next lines